src/components/ShareStructure/ShareStructure.vue

```diff
@@ -149,11 +163,11 @@
             </div>
           </td>
         </tr>
-        <tr v-if="showEditShareStructureForm[row.index]">
+        <tr v-if="showClassEditForm[row.index]">
           <td colspan="6">
             <div class="edit-share-structure-container">
               <edit-share-structure
-                v-show="showEditShareStructureForm"
+                v-show="showClassEditForm"
```

Comment:
Looking at lines 166 and 170, something doesn't seem right. Is `showClassEditForm` an array or a boolean?

Reply:
It is an Array of booleans which are false by default. The index of the array is the index of the row of either the series share or class. The line you highlighted above though, was a redundant check and been removed :)